js-variables/variables.js

```diff
@@ -0,0 +1,22 @@
+// Оголосіть дві змінні adminName та userName. Присвойте цим змінним довільні значення. 
+let adminName = 'Admin';
+let userName = 'User';
+
+// Виведіть тип введених значень з допомогою оператора typeof. 
+console.log(typeof adminName, typeof userName);
+
+// Оголосіть та присвойте змінній значення "awesome".
+let awesome = true;
```

Comment:
оголосіть = let yourVariableName 
присвойте це "="
значення "awesome"
so your variable should look smth like :
```suggestion
let yourVariableName  = "awesome";
```